### ISSUE TITLE
Project submission reminder

### ISSUE THREAD

**Issue by mianzg** [ds4owd-001/project-stacia#2]
@stacianordin Hi there, I have sent a reminder message in your abandoned repo: https://github.com/ds4owd-001/project-stacianordin-delete/issues/3#issuecomment-1929347928. I would like to follow up again because it is now one week behind the original deadline and we have not seen your submission GitHub issue.

Could you give us a submission date (must by Feb 29) if you plan to complete the project? 

Notice that you have to complete the project in order to get a certificate. Please refer to the capstone project requirements here: https://ds4owd-001.github.io/website/project/ and make sure you follow the steps.

**Comment by stacianordin** [comment]
Yes, thank you Mian.  I replied on the Friday email as well and am working on it today.  Stacia

On 19 Feb 2024, at 11:04, Mian ***@***.***> wrote:


@stacianordin <https://github.com/stacianordin> Hi there, I have sent a reminder message in your abandoned repo: ds4owd-001/project-stacianordin-delete#3 (comment) <https://github.com/ds4owd-001/project-stacianordin-delete/issues/3#issuecomment-1929347928>. I would like to follow up again because it is now one week behind the original deadline and we have not seen your submission GitHub issue.

Could you give us a submission date (must by Feb 29) if you plan to complete the project?

Notice that you have to complete the project in order to get a certificate. Please refer to the capstone project requirements here: https://ds4owd-001.github.io/website/project/ and make sure you follow the steps.
